### PR TITLE
fix inferncemodel missing input shape

### PR DESCRIPTION
**Pull request by glorysdj**
this pr is to fix inferncemodel missing input shape
in the future, will change to predict(JTensor) or predict(List<Float>, shape) style

---
This change is [Reviewable](https://reviewable.io/reviews/intel-analytics/analytics-zoo/310)

### CODE REVIEW

**Comment by zhichao-li** [comment]
The conversion looks correct, although it might be a temporally solution.

File: zoo/src/main/java/com/intel/analytics/zoo/pipeline/inference/AbstractInferenceModel.java [hunk 1]
```diff
@@ -47,7 +47,11 @@ public void reload(String modelPath, String weightPath) {
     this.model = InferenceModelFactory.loadFloatInferenceModel(modelPath, weightPath);
   }
 
-  public List<Float> predict(List<List<Float>> input) {
-     return model.predict(input);
+  public List<Float> predict2d(List<List<Float>> input) {
```
Comment (jason-dai) [inline zoo/src/main/java/com/intel/analytics/zoo/pipeline/inference/AbstractInferenceModel.java]: why different names? just use `predict`

**Comment by glorysdj** [comment]
tested in QA ranker.
Result:
2018-05-31 19:10:16 INFO  FloatInferenceModel:36 - predict for input time elapsed [0 s, 178 ms].
[List(-0.015223265, -4.1925344), List(-2.8133392E-5, -10.479748)]

File: zoo/src/main/java/com/intel/analytics/zoo/pipeline/inference/AbstractInferenceModel.java (hunk 1 above)
Comment (jason-dai) [reply zoo/src/main/java/com/intel/analytics/zoo/pipeline/inference/AbstractInferenceModel.java]: Then you may simply define it as `predict(list, dimension)`?

**Comment by zhichao-li** [comment]
I guess it’s because the Java type eraser turn these two methods sharing the same signature

发自我的 iPhone

在 2018年5月31日，下午6:54，Jason Dai <notifications@github.com<mailto:notifications@github.com>> 写道：


@jason-dai commented on this pull request.

________________________________

In zoo/src/main/java/com/intel/analytics/zoo/pipeline/inference/AbstractInferenceModel.java<https://github.com/intel-analytics/analytics-zoo/pull/310#discussion_r192060570>:

> @@ -47,7 +47,11 @@ public void reload(String modelPath, String weightPath) {
     this.model = InferenceModelFactory.loadFloatInferenceModel(modelPath, weightPath);
   }

-  public List<Float> predict(List<List<Float>> input) {
-     return model.predict(input);
+  public List<Float> predict2d(List<List<Float>> input) {


why different names? just use predict

―

File: zoo/src/main/java/com/intel/analytics/zoo/pipeline/inference/AbstractInferenceModel.java (hunk 1 above)
Comment (glorysdj) [reply zoo/src/main/java/com/intel/analytics/zoo/pipeline/inference/AbstractInferenceModel.java]: ok

**Comment by glorysdj** [comment]
PR Validation passed. http://172.168.2.101:8080/view/ZOO-PR/job/ZOO-PR-Validation/537/

**Pull request merged**